tasks-schema.json

```diff
@@ -356,24 +466,175 @@
+                      "type": "object",
+                      "properties": {
+                        "type_id": {
+                          "description": "Object containing required and optional arrays specifying valid sample values.",
+                          "example": [
+                            [1, 2, 3, 4]
+                          ],
+                          "type": "object",
+                          "properties": {
+                            "required": {
+                              "description": "Array of unique sample indexes that must be present for submission to be valid. Can be null if no sample indexes are required and all valid sample indexes are specified in the optional property.",
+                              "type": ["array", "null"],
+                              "items": {
+                                "type": "integer",
+                                "minimum": 1
+                              }
+                            },
+                            "optional": {
+                              "description": "Array of valid but not required unique sample indexes. Can be null if all sample indexes are required and are specified in the required property.",
+                              "type": ["array", "null"],
+                              "items": {
+                                "type": "integer",
+                                "minimum": 1
+                              }
+                            }
+                          },
+                          "required": [
+                            "required",
+                            "optional"
+                          ]
+                        },
+                        "value": {
+                          "type": "object",
+                          "description": "Object defining valid values of samples from the predictive distribution for a given sample index.  Depending on the Hub specification, samples with the same sample index (specified by the type_id) may be assumed to correspond to a joint distribution across multiple levels of the task id variables. See Hub documentation for details.",
+                          "properties": {
+                            "type": {
+                              "description": "Data type of sample value from the predictive distribution.",
+                              "example": "double",
+                              "enum": [
+                                "numeric", "integer", "double"
+                              ]
+                            },
+                            "minimum": {
+                              "description": "The minimum inclusive valid sample value from the predictive distribution",
+                              "example": 0,
+                              "type": "number",
+                              "minimum": 0,
+                              "maximum": 1
```

Comment:
I think there is no general minimum or maximum or the minimum or maximum properties? 🤯 
```suggestion
```

Reply:
So I was going by the definition of the value column for a sample in the googledoc and assuming it was a probability but I may have misunderstood?:
> Numeric: a sample from the predictive distribution.
>
> Note: Depending on the Hub specification, samples with the same sample index (specified by the type_id) may be assumed to correspond to a joint distribution across multiple levels of the task id variables. This is discussed more below.

This maybe related to your comment on whether probability specifications should be just fixed to minimum 0 and maximum 0 rather that a minimum/maximum contraint for each of the properties (which I agree does not make much sense).

Reply:
Ah ok! I seem to have misunderstood the comment all together! You mean we should not make it required?